src/components/LocalFooter/LocalFooter.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Paragraph, Heading, Text } from '../Typography';
+import { Container } from '../Container';
+import { Grid } from '../Grid';
+import { Logo } from '../Logo';
+import { CtaLink } from '../Cta';
+import { schools } from '../../utilities/externalLinks';
+import { initiatives, themes } from '../../utilities/routes';
+
+export const LocalFooter = () => (
+  <Container bgColor="white" py={8}>
+    <Logo isLink className="su-w-300 md:su-w-400 2xl:su-w-[56rem] su-fill-gc-black" />
+    <Grid gap="default" as="nav" aria-label="Local footer" xl={3} pt={7}>
```

Comment:
Using the `nav` landmark with `aria-label` "Local footer". It will be read out as "Local footer navigation" to screen readers.